packages/cashier/src/Containers/deposit.jsx

```diff
@@ -33,7 +33,7 @@ const DepositeSideNote = () => {
         */
     ];
 
-    return <SideNote has_bullets notes={notes} title={<Localize i18n_default_text='Notes' />} />;
+    return <SideNote has_bullets notes={notes} title={<Localize i18n_default_text='Note' />} />;
```

Comment:
Aren't we only changing it to `Note` if there's only one note?

Reply:
That's what I heard too, but the card mentioned to just change the label to Note without conditions specified. I'll confirm if the card description is wrong again or if they decided to change it for all the cases.

Reply:
I confirmed with PM, she says to make it `Note` for all the cases